data/map planets.txt

```diff
@@ -4692,7 +4692,7 @@ planet Violetwake
 planet "Vivid Aether"
 	attributes "gas giant" "requires: gaslining" uninhabited
 	landscape land/nasa29
-	description `Whether it be from local evolution or panspermia, Vivid Aether's clouds are home to colonies of anaerobic microbial life, feeding on both sunlight and traces of methane in the atmosphere. These form colorful streaks in the upper atmosphere, which were what initially attracted Saryd explorers to the planet. Now, it is common for Heliarch research patrols to sit in orbit for weeks at a time in order to study the simple, yet exotic, ecology. Occasionally, they send down hardened research vessels capable of withstanding the extreme pressures, though none happen to be nearby right now.`
+	description `Whether it be from local evolution or panspermia, Vivid Aether's clouds are home to colonies of anaerobic microbial life, feeding on both sunlight and traces of methane in the atmosphere. These form colorful streaks in the upper atmosphere, which were what initially attracted Saryd explorers to the planet. Now, it is common for Heliarch research patrols to sit in orbit for weeks at a time in order to study the simple, yet exotic, ecology. Occasionally, they send down hardened research probes capable of withstanding the pressure, though none stay for long.`
```

Comment:
Sorry, but no, I do not like this suggested change.

Reply:
Instead of saying "sorry I don't like this" and resolving it (several times now), can you please explain yourself and the reasoning?

Reply:
Because the entire point of this is to make the Coalition capabilities  *different* from the Remnant, and not just a straight up copy.

From a lore perspective, the Coalition has a *lot* of space, and a lot of focus on mega-engineering projects. They have zero need to go down.

From a gameplay perspective, the Coalition has a huge number of different directions their story can go and cover, there is zero need or benefit to merely retreading existing story concepts and taking on unique abilities from other factions. 

Beyond both those elements, from a player knowledge perspective, I highly doubt the Heliarchs broadcast their capabilities in this regard, so it seems extremely unlikely to the point of impossibility that the pilot could possibly know any of this. There's no way for them to know it, no way to discover it. Which means that putting this knowledge here is quite simply someone very agressively wanting to shoe horn this into the game.

I am willing to concede mentioning that the Heliarchs have probes they can send into the higher pressure areas, but removing this word makes it more ambiguous about depth. That is the point of removing the word.

Reply:
> Because the entire point of this is to make the Coalition capabilities different from the Remnant, and not just a straight up copy.

It really seems more like you want to insist that their capabilities are *much worse* than the Remnant's.

> From a lore perspective, the Coalition has a lot of space, and a lot of focus on mega-engineering projects. They have zero need to go down.

Lots of people do lots of things, even putting lots of effort into them, that they could be said to have zero need for.
Are they not allowed to research things, try to better understand their environment, just because they want to? Even if not, one could argue that a better understanding of the things around you *is* a necessity.

For that matter, I'm not sure there is actually a demonstrated need for the Remnant to be including gaslining in new ships they are producing.
The Puffins are said to be mothballed when we are given one, suggesting the Remnant aren't routinely investigating gas giants. They may extract resources from gas giants, but that seems to mostly be the job of the carried ships.
None of the new gaslined ships we see really seem to be better suited for this sort of thing anyway, certainly not from a cost point of view, and the Merganser is explicitly said to be intended to take over some of the roles the Starling has, and *it* isn't gaslined.
The station in Esquiline doesn't seem to have been used in a long time until we arrive there in the Cognizance missions. Now, at this point there are reasons for them to be entering gas giants more, but it doesn't seem reasonable that the Merganser and Peregrine were designed with this in mind, given that it's probably not been that long and these don't actually seem to be specialised or otherwise adapted ships that were produced on short notice for this specific task, they are quite clearly general purpose, very good at many things, *especially* combat, the Peregrine is arguably an upgrade over the Albatross in that regard.

To summarise, I think "They have zero need to go down," is a poor argument here.

> From a gameplay perspective, the Coalition has a huge number of different directions their story can go and cover, there is zero need or benefit to merely retreading existing story concepts and taking on unique abilities from other factions.

This doesn't appear to be about gameplay.

> taking on unique abilities from other factions.

While playable, mass produced gaslined starships are currently unique to the Remnant, there is no basis for the idea that the ability to construct craft that can survive deep within a gas giant is as well.
I've also heard about plans, that, as far as I know, you have been generally supportive of, to introduce gaslined starships produced and utilised by the Korath Exiles.

> Beyond both those elements, from a player knowledge perspective, I highly doubt the Heliarchs broadcast their capabilities in this regard,

Why?

> so it seems extremely unlikely to the point of impossibility that the pilot could possibly know any of this. There's no way for them to know it, no way to discover it.

There are numerous things in various descriptions throughout the game that this argument could be applied to. It doesn't seem that the descriptions are necessarily intended to contain information that is always available in character.
Some examples I can immediately think of:
- Any reference to the resistance in Coalition planet descriptions.
- The description of Far'en Lai.
- Various descriptions in Kor Mereti and Sestor space.
- References to the history of the Builders in any planet or outfit descriptions.

> Which means that putting this knowledge here is quite simply someone very agressively wanting to shoe horn this into the game.

That's a very subjective analysis. Your reasoning is unconvincing, and without good reasoning, this description could be applied to any proposed change.

> I am willing to concede mentioning that the Heliarchs have probes they can send into the higher pressure areas, but removing this word makes it more ambiguous about depth. That is the point of removing the word.

Why must it be kept ambiguous? It seems like it's just a matter of: "I don't like that it's there, but you all don't seem to want to let me explicitly say the opposite, so we should just say nothing so I can tell myself it's not true."

Reply:
Alright, then let's make it simple: Revert https://github.com/endless-sky/endless-sky/pull/8274 and let's have the discussion before it's merged.

Reply:
You're going to need to explain that.
It doesn't seem simple.
It seems pretty unreasonable given that you have only expressed concern with this one line.

Reply:
Yes, it does appear that way. Despite that, I'm the one who is supposed to look at the larger picture and decide whether something fits the theme and lore of the game. You know, the whole "theme and lore reviewer" role that no-one actually listens to.

If Arachi wants to discuss this at some future date when they've actually got story to add this and makes use of this, I'm good with talking it over then. Until such time as that happens, this is nothing more, nothing less, than a straight up declaration that someone wants to make gaslining not something special and remove an opportunity for the player to actually get somewhere first for once. There's no reason for it, no justification, no purpose. Just "make gaslining not special" and "ensure the player can't be the first to see this."

Oh, and be a slap in the face for the player that *does* actually make the effort to get here and see this. All that work ,and all they get is a "Yep, Heliarchs got here first."

Reply:
>Yes, it does appear that way. Despite that, I'm the one who is supposed to look at the larger picture and decide whether something fits the theme and lore of the game. You know, the whole "theme and lore reviewer" role that no-one actually listens to.

I think you're mistaken. We went through this before. Yes, you have a lore and thematic role. That does not mean what you say goes. What you say is beneficial when looking at content, yes. But it appears in this case, you are the only one who has a problem with this, meanwhile everyone else finds that there is no issue and there are other ways working around this. It is pretty obvious this is some sort of problem you have with Coalition content, not something as a content reviewer.

Reply:
I'd suggest working with Arachi or others *now*, not continuously saying "No, I don't like this" to any feedback you are getting. That isn't what a lore reviewer does. That is because you are extremely protective about Remnant gaslining and show distaste for anything Coalition related.

Reply:
I think everyone can and should make up their own minds about what input and from who they listen to.
Who gave you the role of "theme and lore" reviewer? As I understand it, you and @MasterOfGrey came up with all of those.

> I'm the one who is supposed to look at the larger picture and decide whether something fits the theme and lore of the game.

This part sounds almost like a direct quote of comments I've heard from @Amazinite relating to his own role.

> Oh, and be a slap in the face for the player that does actually make the effort to get here and see this. All that work ,and all they get is a "Yep, Heliarchs got here first."

If a person considers that a nation of *over 300 billion individuals* with advanced technology has, *checks notes*, investigated a gas giant *in their own territory* before the player, who is using a ship they had to get from someone else, a slap in the face, I think they're rather egocentric.

Reply:
As a matter of fact, I *don't* have a problem with the Coalition.

Arachi and I worked over this once before when dealing. We discussed it, and came to the agreement of de-emphasizing deep exploration into those gasgiants and instead building up the Coalition's industrial, logistics, and engineering capacity in other directions. This made the Coalition more powerful, and also made them more interesting. It added new capabilities not seen anywhere else, and showcased some of the wonders the Coalition's size and capabilities. That was a good example of *positive* additions to the coalition. 

So this change here is basically walking back that previous agreement to go: "You know how we agreed to give coalition megastructures and agreed to not have them spend delving deep into the gasgiants? Scratch that, they need to get everything."

Reply:
Oh, and this is specifically content. You think this is something else? Code, maybe?

Reply:
>Scratch that, they need to get everything."

Many people share this thought with the Remnant as well.

>Oh, and this is specifically content. You think this is something else? Code, maybe?

Regardless of what warp or I work on as a main thing, Derpy has already stated several times that we are able to work on areas outside of that focus as devs and can approach content in general. I find your attempt to be passive aggressive very disheartening. That's like me saying, "Oh, more art for Remnant content? Z's opinion doesn't matter."

Reply:
> Perhaps that was their extent of gaslining so far, and a hard attempt at that.

If that was actually being suggested here, that would be much more viable. But it's not.

The actual counter change proposed very much does seem to be explicitly from directly stepping on the toes of the Remnant, especially as it continues to be insisted on from -outside- of the working together between the two leads that had already happened before it was repeatedly proposed in a way as to be blatantly dismissive.